app/handlers/gate.go

```diff
@@ -92,6 +85,16 @@ func (g gateHandler) PostOpen(c *gin.Context) {
 	c.JSON(http.StatusCreated, content)
 }
 
+func (g gateHandler) createRules(ipNet net.IPNet) []domain.Rule {
+	rules := make([]domain.Rule, len(g.defaultRules))
```

Comment:
Let's check this and all ofter `range` loops on what we are expecting here: https://www.ardanlabs.com/blog/2013/09/iterating-over-slices-in-go.html

Reply:
Can't `range` over a pointer. Also the solution sugested in the blogpost is something a bit too much for this point. We'll be using channels and routines in the near future and this will probably also fix some of the copying